presentations/2025-06-02_replaymeeting.py

```
@@ -0,0 +1,176 @@
+        x_axis = data[0, :]
+
+        if genotype not in to_plot:
+            to_plot[genotype] = [data[1, :]]
+        else:
+            to_plot[genotype].append(data[1, :])
+
+    colors = ["red", "blue", "green"]
+    color_idx = 0
+    for genotype, data in to_plot.items():
+        plt.figure()
+        data_plot = np.array(data)
```

Ruff (0.14.4)

Local variable `data_plot` is assigned to but never used

Remove assignment to unused variable `data_plot`

(F841)